### ISSUE TITLE
The learning rates for actor and critic are different from those in DDPG paper

### ISSUE THREAD

**Issue by maplefox** [yanpanlau/DDPG-Keras-Torcs#34]
In your code:
       actor learning rate is 1e-4
       critic  learning rate is 1e-3
However,in the original DDPG paper which is opposite
       actor learning rate is 1e-3
       critic  learning rate is 1e-4
Could you explain why you make this adjustment,thanks
